Modules/_asynciomodule.c

```diff
@@ -3724,6 +3724,9 @@ module_clear(PyObject *mod)
 
     Py_CLEAR(state->context_kwname);
 
+    _PyThreadStateImpl *ts = (_PyThreadStateImpl *)_PyThreadState_GET();
```

Comment:
Two suggestions:

1. It'd be helpful to have a test that triggers the problematic behavior, if possible.
2. A comment here about why `asyncio_running_loop` is being cleared here (even though it's also cleared in `PyThreadState_Clear()) would also be helpful.